docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/FastForest.cs

```diff
@@ -43,21 +43,21 @@ public static void Example()
                 Console.WriteLine($"Label: {p.Label:F3}, Prediction: {p.Score:F3}");
 
             // Expected output:
-            //   Label: 0.985, Prediction: 0.864
-            //   Label: 0.155, Prediction: 0.164
-            //   Label: 0.515, Prediction: 0.470
-            //   Label: 0.566, Prediction: 0.501
-            //   Label: 0.096, Prediction: 0.138
+            // Label: 0.985, Prediction: 0.864
+            // Label: 0.155, Prediction: 0.164
+            // Label: 0.515, Prediction: 0.470
+            // Label: 0.566, Prediction: 0.501
+            // Label: 0.096, Prediction: 0.138
 
             // Evaluate the overall metrics
             var metrics = mlContext.Regression.Evaluate(transformedTestData);
             Microsoft.ML.SamplesUtils.ConsoleUtils.PrintMetrics(metrics);
```

Comment:
>SamplesUtils [](start = 25, length = 12)

please remove SamplesUtils and just print the metric here with regular console.writeline. we're deprecating SamplesUtils altogether.